test/model/zmq/zmq_common.h

```diff
@@ -15,7 +15,7 @@
 #
 # *******************************************************************************/
 #pragma once
-#include <jsoncpp/json/json.h>
```

Comment:
I think this indeed solves the issue as it previously was using the default included `/usr/include` and going to `jsoncpp/json/json.h` instead of using the included `JSON_INC_PATH` (which resolves to `/usr/include/jsoncpp`) and then going to `json/json.h`.